pir/cpp/client.h

```diff
@@ -47,13 +47,17 @@ class PIRClient {
    * @returns InvalidArgument if the index is invalid or if the encryption fails
    **/
   StatusOr<Request> CreateRequest(std::size_t /*index*/) const;
+  StatusOr<BatchRequest> CreateRequest(
```

Comment:
This code would be a lot simpler if CreateRequest() returned the same time in each case, and the single index case was just a wrapper to call the multiple case with a vector of one item.